src/transform.ts

```diff
@@ -39,7 +40,7 @@ export interface TransformOptions {
 
 export default function transform(options: TransformOptions) {
   // var user = options.user || {};
-  var userId = options.userId;
+  var urlParams = options.urlParams;
   var traits = options.traits || {};
```

Comment:
I think we don't need `traits` in the API any more.

Reply:
@jsedlacek I left it because there's a script in `satismeter` that renders using `traits`: `src/server/lib/email-survey/generate-email-survey.ts`

Do we still use that script?

Reply:
Yes, we use that script, but we can easily adapt the API to the new `urlParams` interface.